nvflare/fuel/utils/decomposers/ml_decomposers.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Decomposers for types from Machine Learning frameworks and libraries."""
+from abc import ABC
+from io import BytesIO
+from typing import Any
+
+import numpy as np
+import tensorflow as tf
```

Comment:
These would make NVFLARE Core depend on np, tf, and torch.

Please move this file to somewhere in app_common.